lib/peep.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# this is the peep class
+class Peep
+  attr_reader :id, :text, :timestamp
+  
+  def initialize(id:, text:, timestamp:)
+    @id = id
+    @text = text
+    @timestamp = timestamp
+  end
+
+  def self.create(text:)
+    result = DatabaseConnection.query("INSERT INTO peeps (text, timestamp) VALUES ('#{text}', '#{Time.now}') RETURNING id, text, timestamp;")
```

Comment:
i also chose to hard code the time into each peep. i think other people went for time methods